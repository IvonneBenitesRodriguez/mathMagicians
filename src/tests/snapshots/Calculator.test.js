import React from 'react';
import { render } from '@testing-library/react';
import CalculatorButton from '../../components/Calculator';

test('Render CalculatorButton with default props', () => {
  const { container } = render(<CalculatorButton />);
  expect(container).toMatchSnapshot();
});

test('Render CalculatorButton with custom props', () => {
  const { container } = render(
    <CalculatorButton
      value="1"
      className="specific-button"
      onClick={() => {}}
    />,
  );
  expect(container).toMatchSnapshot();
});
