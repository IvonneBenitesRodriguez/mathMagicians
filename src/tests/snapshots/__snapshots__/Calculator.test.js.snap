// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CalculatorButton with custom props 1`] = `
<div>
  <button
    class="calculator-button specific-button"
    type="button"
  >
    1
  </button>
</div>
`;

exports[`Render CalculatorButton with default props 1`] = `
<div>
  <button
    class="calculator-button "
    type="button"
  />
</div>
`;
